src/v2/Components/Inquiry/Views/InquiryLogin.tsx

```diff
@@ -146,7 +153,14 @@ export const InquiryLogin: React.FC = () => {
           my={1}
         />
 
-        {mode === Mode.TwoFactor && (
+        {mode === Mode.OnDemand && (
+          <Message mt={2} mb={1}>
+            This login requires additional authorization. Please check your
+            email for a one-time authentication code.
+          </Message>
```

Comment:
**question (non-blocking):** The message component renders a `div`, should we be passing a paragraph tag?

```suggestion
          <Message mt={2} mb={1}>
            <p>This login requires additional authorization. Please check your
            email for a one-time authentication code.</p>
          </Message>
```

Reply:
If we wanted to do this we'd handle it in the Palette level rather than rely on consumers of it to use semantic markup. (and emit an `<aside role="alert"><p>...` etc...). And differentiate between text children and markup children to support multiple paragraphs.

(Which is a good idea, and one I'll ticket in DSWG)